src/qt/clientmodel.cpp

```diff
@@ -177,6 +177,11 @@ QString ClientModel::dataDir() const
     return GUIUtil::boostPathToQString(GetDataDir());
 }
 
+QString ClientModel::blocksDir() const
+{
+    return GUIUtil::boostPathToQString(GetBlocksDir());
```

Comment:
Should not call `GetBlockDir` but get it from `interfaces::Node`? cc @ryanofsky

Reply:
In the long run, promag is right and adding an `interfaces::Node` method would be better, but for now calling `GetBlocksDir()` directly is more consistent with existing code. The goal of adding the `Node` and `Wallet` interfaces in #10244 was to get rid of accesses to non-GUI global variables from GUI code. But to make the change more minimal, I exempted `gArgs` and chainparams variables, and instead added some glue code in #10102 to let each process just keep its own copies of these variables. As a result, it's fine to call functions like `GetBlocksDir` and `GetDataDir` from GUI code that access them.